### DISCUSSION TITLE
Clicking on q-menu, scrolls me to top

### DISCUSSION THREAD

**Discussion by saifobeidat** [quasarframework/quasar#10186] [General - Components / Directives / etc]
When I show the q-menu component, the page gets scrolled to top!

https://user-images.githubusercontent.com/24841143/126898811-a726f410-e03e-4b94-9dc3-009012823923.mp4

I noticed that a scroll event gets fired when I show the q-menu:
![image](https://user-images.githubusercontent.com/24841143/126898854-4d104b6f-66fa-4c0f-bb54-337ec292dc12.png)

How this could happen? as I don't trigger any events when showing q-menu, and I couldn't reproduce it on CodePen

This is what I have
```
 <q-layout view="hHh lpR fFf">
  <q-header>
  <q-btn color="primary" label="Basic Menu">
        <q-menu>
          <q-list style="min-width: 100px">
            <q-item clickable v-close-popup>
              <q-item-section>New tab</q-item-section>
            </q-item>
          </q-list>
        </q-menu>
      </q-btn>
  </q-header>
</q-layout>
```

anyone has insights?  Thanks

**Comment by MilosPaunovic** [comment]
You have `QScrollObserver` somewhere, that is what is trigerring the event.

**Reply by saifobeidat** [reply]
I found the cause, there was this code in the project where it overrides the default styles of q-menu which causes that behavior.. I just removed it, and it worked fine
``` 
.q-menu {
   position: absolute !important;
 }
```
